engine/src/org/pentaho/di/trans/steps/scriptvalues_mod/ScriptValuesAddedFunctions.java

```diff
@@ -2580,7 +2605,7 @@ public static String execProcess( Context actualContext, Scriptable actualObject
         while ( ( ligne = br.readLine() ) != null ) {
           buffer.append( ligne );
         }
-        // if (processrun.exitValue()!=0) throw Context.reportRuntimeError("Error while running " + ArgList[0]);
+        // if (processrun.exitValue()!=0) throw Context.reportRuntimeError("Error while running " + arguments[0]);
```

Comment:
Looks like not, but this comment wasn't brought by me, so I prefered not to touch it.